system/mavros2/mavros.launch.xml

```diff
@@ -0,0 +1,40 @@
+<launch>
+
+    <arg name="vehicle_namespace" default="$(env VEHICLE_NAMESPACE vehicle_$(env MAVROS_TGT_SYSTEM))" />
+    <arg name="firmware" default="$(env MAVROS_TGT_FIRMWARE)" />
+    <arg name="fcu_url" default="$(env MAVROS_FCU_URL)"/>
+    <arg name="gcs_url" default="$(env MAVROS_GCS_URL)" />
+    <arg name="target_system" default="$(env MAVROS_TGT_SYSTEM)" />
+    <arg name="system_id" default="$(env MAVROS_TGT_SYSTEM)" />
+    <arg name="config_yaml" default="$(env MAVROS_MOD_CONFIG_PATH)"/>
+    <arg name="pluginlists_yaml" default="$(env MAVROS_MOD_PLUGINLISTS_PATH)"/>
+   	<arg name="tgt_component" default="1" />
+    <arg name="respawn_mavros" default="true"/>
+    <arg name="log_output" default="screen" />
+	<arg name="fcu_protocol" default="v2.0" />
+
+    <node pkg="mavros"
+          exec="mavros_node"
+          namespace="$(var vehicle_namespace)/mavros">
+        <param name="fcu_url" value="$(var fcu_url)?ids=$(var system_id),240"/>
+        <param name="gcs_url" value="$(var gcs_url)"/>
+        <param name="tgt_system" value="$(var target_system)"/>
+        <param name="tgt_component" value="$(var tgt_component)" />
+
+        <param from="$(var config_yaml)"/>
+        <param from="$(var pluginlists_yaml)"/>
+
+        <!-- <remap from="set_mode" to="mavros/set_mode"/> -->
+        <!-- <remap from="set_stream_rate" to="mavros/set_stream_rate"/> -->
+        <!-- <remap from="set_message_interval" to="mavros/set_message_interval"/> -->
+    </node>
+
+    <!-- Launch Estop Node -->
+    <node name="estop" pkg="starling_mavros_extras" exec="estop" output="screen" respawn="true" namespace="$(var vehicle_namespace)"/>
```

Comment:
While convenient to have in the mavros container, is it appropriate? I see this as more starling specific rather than plain mavros. Overhead of a whole container is probably a bit much though...

Reply:
I think it is appropriate as the Mavros container is basically the defacto core container. Therefore core functionality should be included. We want the ESTOP to be everywhere if possible so we keep it in. Same with the ping mechanic.